dart/biomechanics/DynamicsFitter.cpp

```diff
@@ -10443,11 +10440,11 @@ void DynamicsFitter::zeroLinearResidualsOnCOMTrajectory(
       // and total mass. It outputs the physically consistent X, Y, Z
       // coordinates of COM over time, given those inputs. Additionally, there
       // are columns to account for small force plate rotation errors (3 angles
-      // per force plate) and for instantaneous COM velocities at time steps with
-      // missing ground reaction force data.
+      // per force plate) and for instantaneous COM velocities at time steps
+      // with missing ground reaction force data.
       Eigen::MatrixXs trialLinearMapToPositions = Eigen::MatrixXs::Zero(
-          numTimesteps * 3 + (numForcePlates * 3) + (numMissingSteps * 3),
-          6 + (numForcePlates * 3) + (numMissingSteps * 3) + 1);
+          numTimesteps * 3 + (numForcePlates * 3) + (numMissingSteps * 3) + 1,
```

Comment:
Here I add a column for the new "scaleGRF" value, and I add a row to regularize it.